ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/mac-security.j2

```diff
@@ -37,10 +37,10 @@ mac security
 {%         if profile.sci is arista.avd.defined(true) %}
       sci
 {%         endif %}
-{%         if profile.l2_protocols.ethernet_flow_control is arista.avd.defined %}
+{%         if profile.l2_protocols.ethernet_flow_control.mode is arista.avd.defined %}
       l2-protocol ethernet-flow-control {{ profile.l2_protocols.ethernet_flow_control.mode }}
 {%         endif %}
-{%         if profile.l2_protocols.lldp is arista.avd.defined %}
+{%         if profile.l2_protocols.lldp.mode is arista.avd.defined %}
```

Comment:
same here, mode is required key

Reply:
required keys are not strictly enforced currently. So for now we need these checks.